open_learning_ai_tutor/utils.py

```diff
@@ -10,7 +10,7 @@
 )
 
 
-def tutor_output_to_json(chat_history, intent_history, assessment_history, metadata):
+def tutor_output_to_json(chat_history, intent_history, assessment_history, metadata={}):
```

Comment:
its probably good to avoid using empty lists/dicts as default arguments. maybe something like this instead:
```python
def tutor_output_to_json(chat_history, intent_history, assessment_history, metadata={}):
    metadata = metadata or {}
```

Reply:
i made the requested change